.github/dependabot.yml

```diff
@@ -1,12 +1,11 @@
 version: 2
 updates:
-- package-ecosystem: pip
-  directory: "/"
-  schedule:
-    interval: daily
-  open-pull-requests-limit: 10
-- package-ecosystem: github-actions
-  directory: /
-  schedule:
-    interval: daily
-  open-pull-requests-limit: 10
+  - package-ecosystem: pip
```

Comment:
Why did you want to add extra indents? Here's the style I tend to enforce through linting: https://github.com/aio-libs/aiomonitor/blob/9ec9910145c70037da8ab52e007b9fc286c3a4c7/.yamllint.

Reply:
I was copying the config from aiohttp-jinja2, which I generally use as my reference repo. I think we should just focus on ways to remove the config from these repos. The `uses` approach you've mentioned above looks like it could work well with my submodule idea to include base configs into all repos that can be managed in one place.

Reply:
Since I have projects outside this org, some of my maintenance experiments happen there too. So for me, that's not a reference I'd approve of normally. I already have a vision and some plans for making things easier to maintain, but not enough hours in a day. Though, I'm linking aiomonitor and aiomysql since their current maintainers have been using the references I wanted and provided them with, to some extent.
We do need to remove the boilerplate, but probably not the core configuration. Having introduced a git submodule-based approach in aiohttp years ago, I now don't think that this is the best way forward. Now, I lean towards something similar to git subtrees and/or actually merging the trees in, like what https://github.com/jaraco/skeleton does.

Reply:
I'm just trying to think of the easiest way to maintain. I figured a submodule could be automatically updated by Dependabot to pick up the newest configs. If changes are needed for the new config, the PR will fail CI and require someone to update.

I don't think you can use Dependabot with those other solutions.

Reply:
I'm almost sure dependabot can't work with git modules. But even if it did, this would still bring a lot of friction for the contributors because of the need for every person to remember to synchronize the submodules additionally.

Reply:
> I'm almost sure dependabot can't work with git modules.

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem
`gitsubmodule`